pages/restake/bridge.mdx

```diff
@@ -0,0 +1,78 @@
+import Callout from "/components/Callout";
+
+# Bridge
+
+In order to participate in Tangle's restaking infrastructure, users need to first bridge in their assets from EVM-based networks such as Ethereum into Tangle. For this, we have a dedicated bridge & its corresponding DApp page.
+
+[Access Tangle DApp's Bridge page here](https://app.tangle.tools/bridge)
+
+## Supported Assets
+
+As of the time of writing of this page, the bridge only supports WETH for testnet transfers between Holesky and Tangle.
+
+Soon, users will be able to create and deploy a Hyperlane Warp Route blueprint instance to support additional assets. [Learn more about Blueprints here](/developers/blueprints).
+
+More information about adding support for additional assets will be provided in the near future. For now, make sure you've joined our Discord server & Telegram channel to stay updated.
+
+TODO: For custom bridging, how will the users choose what bridge to use if anyone can deploy their warp route instances? The list will need to be of addresses, but those addresses will need to be registered in a specific repo. We don't have that repo yet, so for now we depend on the Hyperlane registry repo in which users can submit their PRs to have their own instances included.
+
+## How the Bridge Works
```

Comment:
We will support many bridges likely, we should just have a list of supported bridges and assets and keep it updated. You don't need to then discuss things so specifically but rather generally.